src/main/java/com/google/devtools/build/lib/runtime/UiOptions.java

```diff
@@ -178,6 +234,17 @@ public ProgressModeConverter() {
       help = "Report all events known to the Bazel UI.")
   public boolean experimentalUiDebugAllEvents;
 
+  @Option(
+      name = "ui_event_filters",
+      converter = EventFiltersConverter.class,
+      defaultValue = "null",
+      metadataTags = {OptionMetadataTag.HIDDEN},
```

Comment:
Let's not make this flag hidden (this would only be appropriate for experimental flags or flags that are not meant to be specified by a human user).